getstarted.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default-nav
+title: Get Started
+description: mlpack | fast, flexible machine learning library in C++
+order: 2
+---
+
+<div class="page-title-header">Get Started</div>
+
+<h1>Overview</h1>
+
+There are multiple ways to get mlpack up and running. Python bindings can be installed using pip or conda, or built from source (see "Build from source" tutorials).
+For C++, if mlpack is not available via your preferred OS package manager, or if you need to build your own version (e.g. to apply optimizations, use a different 
+set of BLAS/LAPACK, or build a different configuration), please refer to the "Build from source" tutorials.
+For Windows, prebuilt binaries will help you get started without the need of building mlpack. These packages include both the C++ mlpack library as well as the CLI tools.
+
+Here is a summary of the currently available distribution options you can use depending on your needs:
+
+<h2>Python</h2>
+
+<h3> Get via pip or conda </h3>
+- <a href="doc/mlpack-3.1.0/doxygen/python_quickstart.html"> mlpack in Python quickstart guide </a>
+
+<h2> Ubuntu, Debian, Fedora, Red Hat </h2>
+
+<h3> Install using a package manager </h3>
+- <a href="doc/mlpack-3.1.0/doxygen/cli_quickstart.html"> mlpack command-line quickstart guide </a>
+
+<h3> Build from source </h3>
+- <a href="doc/mlpack-3.1.0/doxygen/build.html"> Linux build tutorial </a>
+
+<h2> Windows </h2>
+
+<h3> Download prebuilt binaries </h3>
+- <a href="/files/mlpack-3.1.0.msi"> Windows 64 bit - MSI Installer </a>
+- <a href="/files/mlpack-3.1.0.zip"> Windows 64 bit - ZIP </a>
+
+<h3> Install using a package manager </h3>
+- <a href="doc/mlpack-3.1.0/doxygen/build_windows.html"> Installation through vcpkg </a>
+
+<h3> Build from source </h3>
+- <a href="doc/mlpack-3.1.0/doxygen/build_windows.html"> Windows build tutorial </a>
```

Comment:
I agree, although the "simple card" that we have now doesn't work for the purpose (just incompatible CSS). I believe a "big/long card" version would be required, so the card can summarize the content in itself (instead of adding more clicks to the UX, which is not recommended).
Let's add an additional story to work on that separately, as I can't get it reworked today.